packages/currency/src/conversion-aggregators.ts

```diff
@@ -48,6 +48,7 @@ const noConversionNetworks: AggregatorsMap = {
   xdai: {},
   avalanche: {},
   bsc: {},
+  optimism: {},
```

Comment:
can we change something to avoid needing this?

Reply:
I think we can (and should) but it needs to be thought about, it's not an easy change iirc